app/commands/solution/mentor_request/retrieve_tracks.rb

```diff
@@ -3,9 +3,8 @@ class MentorRequest
     class RetrieveTracks
```

Comment:
I removed `selected_id` here. We handle passing the selected track for the mentoring queue differently now.